dags/veda_data_pipeline/groups/collection_group.py

```diff
@@ -37,13 +37,13 @@ def ingest_collection_task(ti=None, collection=None):
         collection = ti.xcom_pull(task_ids='Collection.generate_collection')
     airflow_vars = Variable.get("aws_dags_variables")
```

Comment:
We can add `deserialize_json` parameter so we will not have to call `json.loads` 
```suggestion
    airflow_vars_json = Variable.get("aws_dags_variables", deserialize_json=True)
```